build.xml

```diff
@@ -59,14 +381,72 @@
 		</exec>
 	</target>
 
-	<!-- to option copy file to webapps :) -->
+	<target name="clear-tomcat" description="Delete Tomcat workdir">
+		<delete dir="${env.CATALINA_HOME}/work/Catalina/localhost"/>
+		<delete file="${env.CATALINA_HOME}/webapps/ScadaBR.war"/>
+		<delete dir="${env.CATALINA_HOME}/webapps/ScadaBR"/>
+	</target>
+
+	<target name="build" description="Perform project build workflow" depends="
+		init-core,
+		init-static-dependencies,
+		init-ui-dependencies,
+		build-core,
+		build-ui
+	"/>
+
+	<target name="build-no-ui" description="Build just a core application" depends="
+		init-core,
+		init-static-dependencies,
+		build-core"/>
+
+	<target name="deploy" description="Deploy application to Tomcat workdir" depends="
+		clear-tomcat,
+		deploy-war,
+		deploy-tomcat"/>
+
+	<target name="run-build" description="Build and run application" depends="
```

Comment:
build-deploy-run

Reply:
If you want to build and run application that is obvious that you have to deploy it. So `deploy` is unnecesary

Reply:
You're right, deploy is redundant.